test_genie_python_dae.py

```diff
@@ -654,3 +655,30 @@ def test_GIVEN_x_seconds_have_elapsed_since_start_WHEN_getting_time_since_start_
         # Taking the fluctuation of actual runtime into account and tolerating up to 1 sec difference
         self.assertAlmostEqual(expected, actual_s, delta=tolerance)  # if this fails, then will print the two values
         self.assertTrue(abs(timedelta(seconds=expected) - actual_timedelta) < timedelta(seconds=tolerance))
+
+    def test_GIVEN_no_instetc_WHEN_get_rb_and_begin_run_THEN_dae_state_is_running(self):
+        """
+        Checks that if INSTECT is down and methods like get_rb_number timeout/fail, then a user can begin a run
+        and the DAE functions as expected without getting stuck.
+        """
+        timeout = 30
+
+        g.waitfor_runstate("SETUP", maxwaitsecs=timeout)
+
+        # Turn off INSTETC
+        stop_ioc("INSTETC_01")
+        wait_for_ioc_start_stop(timeout, False, "INSTETC_01")
+
+        # Try to get the RB number (should timeout)
+        self.assertRaises(Exception, g.get_rb())
```

Comment:
I think this needs to be either:

```suggestion
        self.assertRaises(Exception, g.get_rb)
```

or 

```suggestion
        with self.assertRaises(Exception):
            g.get_rb()
```